lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -2068,18 +2080,19 @@ def jsonrpc_claim_list(self, account_id=None, wallet_id=None, page=None, page_si
 
         Returns: {Paginated[Output]}
         """
+        async def claims(**kwargs):
+            return (await self.ledger.claim_search(**kwargs))[0]
+        async def claim_count(**kwargs):
+            return (await self.ledger.claim_search(**kwargs))[2]
```

Comment:
`claim_search` is an extremely expensive operation (relatively speaking), calling it a second time to get the count which is already available doesn't make any sense.

Reply:
@eukreign is the `--no-totals` parameter still needed then? if we make only one call, we can always return totals and it will not impact the performance

Reply:
The performance penalty is on the server side, not on the client or in the communication stack.